scripts/globals/mobskills/nuclear_waste.lua

```diff
@@ -0,0 +1,35 @@
+---------------------------------------------------
+--  Nuclear Waste
+--  Description: Reduces elemental resistances by 50 to players in range.
+---------------------------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/monstertpmoves")
+
+---------------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    -- skillList  54 = Omega
+    -- skillList 727 = Proto-Omega
+    -- skillList 728 = Ultima
+    -- skillList 729 = Proto-Ultima
+    local skillList = mob:getMobMod(dsp.mobMod.SKILL_LIST)
+    local mobhp = mob:getHPP()
+    local phase = mob:getLocalVar("battlePhase")
+
+    if ((skillList == 729 and phase >= 1 and phase <= 2) or (skillList == 728 and mobhp < 70 and mobhp >= 40)) then
+        if mob:getLocalVar("nuclearWaste") == 0 then
+            return 0
+        end
+    end
+
+    return 1
+end
+
+function onMobWeaponSkill(target, mob, skill)
+    mob:setLocalVar("nuclearWaste", 1)
+    local typeEffect = dsp.effect.ELEMENTALRES_DOWN
+    target:addStatusEffectEx(dsp.effect.ELEMENTALRES_DOWN, 0, 50, 0, 60)
+    skill:setMsg(msg)
```

Comment:
`msg` is nil in this context. I understand why you didn't use `MobStatusEffectMove()` but you need to add the logic for what message should be set here (and anything else we would normally be doing that this should use) -or- just modify the function to let you use the subid and Ex version for adding your status effect.

Reply:
oh whoops this was a last minute change. It should be giving no message and at one point was set to dsp.msg.basic.none but I was messing around with different ways to do this and ended up with just making it 100% apply for now and forgot to change the message back.

Reply:
okies :+1: 

I resume there is also no resist calc to be done. so if there is no message at all with it, that'll do.

Reply:
It actually should be able to resist so I'm gonna have to edit the MobStatusEffectMove function.

Reply:
a reist calc is small enough you could do it here if you want, there a function for the resist check that can be tossed to a variable, then you just check the resist value against the target tier or use it as a modifier (depending on what resistance does for this particular effect.

It's in the form of a float value because dsp's resist system is convoluted af.

For hit or mis style or resists its:
```lua
    local resist = applyPlayerResistance(whatever);
    if resist >= 0.25 then
        --stuff
    end
```
for reduced duration we have:
```lua
    local resist = applyPlayerResistance(whatever)
    local totalDuration = utils.clamp(duration * resist, 1);
```

and for dmg reduction we have:
```lua
    local resist = applyPlayerResistance(whatever)
    dmg = dmg * resist
```

Reply:
if you do choose to edit MobStatusEffectMove move tho that will help future additions

Reply:
Awesome, thanks! I'm pretty sure it's hit or miss. It doesn't do any damage and the duration I'm using right now is completely made up anyway since it's pretty hard to see how this effect actually works. I'm even clearing it immediately if you get hit by the elemental ability that comes next.

Reply:
Just in case, see if `handleNinjutsuDebuff()` already does what you need

Reply:
That doesn't look like I can use it to lower all elements at the same time, also doesn't seem to resist at all.

Reply:
well in theory you'd just call it for each element. but if you need a resist check thats no good.